src/coreclr/jit/emitxarch.cpp

```diff
@@ -758,19 +758,29 @@ bool emitter::DoesWriteSignFlag(instruction ins)
 }
 
 //------------------------------------------------------------------------
-// DoesResetOverflowAndCarryFlags: check if the instruction resets the
-//     OF and CF flag to 0.
+// DoesModifySameFlagsAsTESTInst: check if the instruction modifies
+//    the same flags as a "TEST" instruction, which allows some optims
```

Comment:
Can we provide more detail on what it allows, if it's not too verbose?

Reply:
Hi @kg thans for the review, very appreciated 🙂 
Yes sure I can add more info and point out that this allows to potentially skip a test instruction.